proposals/0000-borrow-operator.md

```diff
@@ -0,0 +1,189 @@
+# Borrow Operator, Pass-by-Borrow, and Pass-by-Value
+
+* Proposal: [SE-NNNN](NNNN-filename.md)
+* Authors: ...
+* Review Manager: TBD
+* Status: ...
+
+<!--
+
+*During the review process, add the following fields as needed:*
+
+* Implementation: [apple/swift#NNNNN](https://github.com/apple/swift/pull/NNNNN) or [apple/swift-evolution-staging#NNNNN](https://github.com/apple/swift-evolution-staging/pull/NNNNN)
+* Decision Notes: [Rationale](https://forums.swift.org/), [Additional Commentary](https://forums.swift.org/)
+* Bugs: [SR-NNNN](https://bugs.swift.org/browse/SR-NNNN), [SR-MMMM](https://bugs.swift.org/browse/SR-MMMM)
+* Previous Revision: [1](https://github.com/apple/swift-evolution/blob/...commit-ID.../proposals/NNNN-filename.md)
+* Previous Proposal: [SE-XXXX](XXXX-filename.md)
+
+-->
+
+## Introduction
+
+In this document, we propose adding a new operator to the language, marked by
+the context sensitive keyword `borrow`, that causes the compiler to pass
+arguments using a `pass-by-borrow` convention instead of a `pass-by-value`
```

Comment:
I was purposely using this since I want it to be viewed as a key word for the reader to pay attention to. Similar to the way that JoeG highlighted borrow and take in the following proposal: https://github.com/apple/swift-evolution/blob/main/proposals/0377-parameter-ownership-modifiers.md